app/views/public_mailer/submission_confirmation_et.html.slim

```diff
@@ -0,0 +1,84 @@
+table cellpadding="0" cellspacing="0" border="0" width="600"
```

Comment:
Someone had to do it. HTML email is never pretty, but sometimes it just has to be done.

Good work soldier. Carry on. 👍